instant-kraft/README.md

```diff
@@ -0,0 +1,56 @@
+# Instant KRaft Demo
+Let's make some delicious instant KRaft
+          ___
+        .'O o'-._
+       / O o_.-`|
+      /O_.-'  O |
+      |O o O .-`
+      |o O_.-'
+      '--`
+
+## Step 1
+```
+$ git clone https://github.com/confluentinc/examples.git && cd examples/instant-kraft
+
+$ docker compose up
+```
+
+That's it! One container, one process, doing the the broker and controller work. No Zookeeper.
+The key server configure to enable KRaft is the `process.roles`. Here we have the same node doing both the broker and quorum conroller work. Note that this is ok for local development, but in production you should have seperate nodes for the broker and controller quorum nodes.
```

Comment:
Needs rewording.

The key server configuration for enabling KRaft is `process.roles`.